pkg/tournament/classic_division.go

```diff
@@ -952,6 +955,7 @@ func (t *ClassicDivision) RemovePlayers(persons *realtime.TournamentPersons) (*r
 				return nil, err
 			}
 			pairingsMessage.DivisionStandings = combineStandingsResponses(pairingsMessage.DivisionStandings, map[int32]*realtime.RoundStandings{int32(i): roundStandings})
+			t.Standings[int32(i)] = roundStandings
```

Comment:
Oops. This suggestion that was mine doesn't actually fix anything. I've messaged you with the REAL answer.